_posts/2024-10-24-gift-message.md

```diff
@@ -0,0 +1,138 @@
+
+![Order note example](/img/blog/gift-note/order-notes.png)
+
+This has a couple of drawbacks for a flower shop:
+
+- This is not intuitive for the customer. You often need to add extra instructions to tell the customer when and where they can write a gift note.
+- If there are multiple arrangements in an order, there will still only be one cart note.
+- This box is not just used for gift notes but desi45gn preferences and other instructions.
+
+In this post, we’ll go over how to add a gift note to individual products to give customers a better experience. By the end of this blog, you'll have something like this on your product pages:
+
+![Gift note example](/img/blog/gift-note/gift-note-for-theme.png)
+
+It's quite simple and we can do it without any apps. You will need a little code, but we’ll walk you through it. **Note that we are using Google Chrome for our web browser, which is relevant for Step 3.**
+
+## Step 1: Find the Product page in your theme editor
+
+Navigate to the “Customize” button of your theme editor on the “Themes” screen
+
+![Open theme editor](/img/blog/gift-note/theme-editor.png)
+
+Navigate to your product page:
+
+![Navigate to your product page](/img/blog/gift-note/product-nav.png)
+
+## Step 2: Create the gift note block on the page
+
+Find the section that says "Product Information." Click "Add block," and choose "Custom Liquid."
+
+![Add block](/img/blog/gift-note/add-block.png)
+
+Add this custom liquid code:
+
+```html
+<textarea id="giftnote" placeholder="Gift Note" name="properties[giftnote]"></textarea>
+```
+
+Drag and drop your “Custom Liquid” block to the desired place on your product page. We’ve added it after the Quantity selector below.
+
+![Custom liquid](/img/blog/gift-note/custom-liquid.png)
+
+Yay, we’ve added a Gift Note to the page! We still need to link it to the order form so that the note is linked to the product when it’s added to the cart.
+
+## Step 3: Link the gift note to the order form
+
+Now we have to edit the “Custom Liquid” block you just made so it links to the form. To do this, we need to find the generic form ID for your theme.
+
+If you’re using **Dawn** or **Sense**, then you can luckily skip this step! We've found the generic form ID, so you can update your "Custom Liquid" with the attribute `form="product-form-{{ section.id }}"` so it looks like this:
+
+```html
+<textarea id="giftnote" placeholder="Gift Note" name="properties[giftnote]" form="product-form-{{ section.id }}"></textarea>
+```
+
+Then skip to Step 4. If you're using a different theme, let's find the generic form ID together.
+
+### Find the theme's generic form ID:
+
+In a separate window, open a product page on your website. Right-click on your “Add to Cart” button and select "Inspect."
+    ![Inspect Add to Cart button](/img/blog/gift-note/inspect-add-to-cart.png)
+This will open the Web Inspector. Don’t be intimidated, we’ll go over it together. We’re going to look at a couple of elements:
```

Comment:
```suggestion
This will open Google Chrome's Web Inspector. Don’t be intimidated, we’ll go over it together. We’re going to look at a couple of elements:
```